### PR TITLE
Develop UI RPC server infrastructure.

### PR DESCRIPTION
**Pull request by ababo**
This commit contains the following changes:
1. RPC server boilerplate code in `util/rpcsrv`.
2. UI handler with implemented AcceptOffering method in `ui`.
3. Enhancements for the new logger (Add() supports multiple vars).
4. job.Queue has been abstracted to an interface to simplify testing.
5. Added to job.Queue support for related ID subscriptions.
6. Fixed AcceptOffering flow to have a same RelatedID.

Resolves BV-590.

### CODE REVIEW

File: ui/handler.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package ui
+
+import (
+	"github.com/privatix/dappctrl/job"
+	"github.com/privatix/dappctrl/util/log"
+	"github.com/privatix/dappctrl/util/rpcsrv"
+
+	"gopkg.in/reform.v1"
```
Comment (furkhat) [inline ui/handler.go]: internal packages goes last

File: ui/handler_test.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	"gopkg.in/reform.v1"
+
+	"github.com/privatix/dappctrl/data"
+	"github.com/privatix/dappctrl/job"
+	"github.com/privatix/dappctrl/util"
+	"github.com/privatix/dappctrl/util/log"
+)
+
+const testSalt = "test-salt"
+const testPassword = "test-password"
```
Comment (furkhat) [inline ui/handler_test.go]: could you use `data. TestPassword` and `data. TestSalt` instead?

File: ui/misc.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+	"gopkg.in/reform.v1"
+
+	"github.com/privatix/dappctrl/data"
+	"github.com/privatix/dappctrl/util/log"
+	"github.com/privatix/dappctrl/util/rpcsrv"
+)
+
+const (
+	passwordKey = "system.password"
+	saltKey     = "system.salt"
```
Comment (furkhat) [inline ui/misc.go]: We should move settings constant to `data` package

**Comment by ababo** [comment]
@furkhat updated

**Pull request merged**